kdl_parser_py/package.xml

```diff
@@ -30,10 +30,10 @@
   <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-catkin-pkg</buildtool_depend>
 
   <build_export_depend>urdfdom_py</build_export_depend>
-  <build_export_depend>python_orocos_kdl</build_export_depend>
+  <build_export_depend>python3-pykdl</build_export_depend>
```

Comment:
Mind deleting this line? It doesn't look like a `<build_export_depend>` is necessary; the `<exec_depend>` below should be enough.